pkg/backend/openshiftcluster/create.go

```diff
@@ -306,13 +306,23 @@ func (m *Manager) Create(ctx context.Context) error {
 	}
 
 	image := &releaseimage.Image{}
-	if m.doc.OpenShiftCluster.Properties.ClusterProfile.Version == version.OpenShiftVersion {
-		image.PullSpec = version.OpenShiftPullSpec
+	v, err := version.ParseVersion(m.doc.OpenShiftCluster.Properties.ClusterProfile.Version)
```

Comment:
This is a create flow, not update.  I'm not expecting this PR to enable a single RP to be able to create different versions of cluster -- are you?

Reply:
This is already blocked by frontend code. This just prevents stale documents... Might not be best idea. Need to discuss.

Reply:
Rollback